js/data/locale/und/DE/numplan.json

```diff
@@ -1,10 +1,9 @@
 {
     "region": "DE",
-    "countryCode": "49",
```

Comment:
In all of these ones that are modified, the country code is missing.

Reply:
Oh yes. That part was I manually updated. Because the tool code did not generate information for the country, and the existing version of files did not have a country code before... I'll update again